org.technic93.e2designer.yaml

```diff
@@ -0,0 +1,20 @@
+app-id: "org.technic93.e2designer"
+runtime: org.kde.Platform
+runtime-version: '5.14'
+sdk: org.kde.Sdk
+command: e2designer
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --filesystem=host
+cleanup:
+  - /usr
+modules:
+  - name: e2designer
+    buildsystem: qmake
+    builddir: true
+    sources:
+      - type: git
+        url: https://gitlab.com/technic93/e2designer/-/archive/v0.2/e2designer-v0.2.tar.gz
+        branch: master
```

Comment:
A file needs a sha256 and not a branch

Reply:
What's the best way to get it? from git or from sha256?